server.js

```diff
@@ -1944,6 +1948,14 @@ if (config.startServer) {
           callback(null);
         });
       },
+      async () => {
+        compiledAssets.init({
+          dev: config.devMode,
```

Comment:
The fact that the `prairielearn/prairielearn` Docker image runs in dev mode is unfortunate, as it means we won't be taking advantage of prebuilt assets. esbuild should be fast enough where this shouldn't be a big issue, but there's room for improvement here. A potential option is that we should produce two images, e.g. `prairielearn/prairielearn` and `prairielearn/prairielearn-dev`?

Reply:
I'm on board with a separate `-dev` image for people doing development. It would certainly be nice to keep `prairielearn/prairielearn` as fast as possible.

Reply:
Absolutely. This would also help when we deploy `prairielearn/executor`, which doesn't need to include many of the tools that a development-focused image would.